src/server/scripts/Northrend/Naxxramas/instance_naxxramas.cpp

```diff
@@ -477,6 +480,17 @@ class instance_naxxramas : public InstanceMapScript
                                 kelthuzad->AI()->Talk(CurrentWingTaunt);
                             ++CurrentWingTaunt;
                             break;
+                        case EVENT_SUMMON_FROGGER_WAVE:
+                        {
+                            static const uint32 FROGGER_ENTRY = 16027;
```

Comment:
Why not. I considered putting it just outside the method, but it looks weird there, too.